peer.go

```diff
@@ -313,6 +331,21 @@ func (p *peer) Start() error {
 			"must be init message")
 	}
 
+	// If we had the interaction with this peer before than we should
+	// retrieve the messages that were not acked in previous session and
+	// sent them again in order to be sure that remote peer is handled them.
+	messages := p.retransmitter.MessagesToRetransmit()
+	if len(messages) != 0 {
+		peerLog.Infof("retransmission subsystem resends %v messages "+
+			"to the peer(%v)", len(messages), p)
+
+		for _, message := range messages {
+			// Sending over sendToPeer will cause block because of
```

Comment:
Can you insert a logging message here that just logs the `MessageCode` itself? Thanks!